server.js

```diff
@@ -0,0 +1,75 @@
+import mongoose from "mongoose";
+import dotenv from "dotenv";
+import express from "express"
+import bodyParser from "body-parser"
+import cors from "cors"
+import validator from 'validator';
+
+dotenv.config({ path: "./config.env" });
```

Comment:
The most common practice is to put environment variables in `.env` (hence the name of the package that reads them).